### ISSUE TITLE
ClayMultiSelect: Focus

### ISSUE THREAD

**Issue by drakonux** [liferay/clay#1526]
**Steps to Reproduce**
1. Click an empty Multiselect

*Current behavior*
Multiselect has no focus style

![not focused](https://user-images.githubusercontent.com/21216422/52572652-9414e180-2e18-11e9-9869-02cee82474d1.gif)

*Expected behavior*
Multiselect should have a focus indicator
